src/test/java/seedu/cafectrl/command/AddOrderCommandTest.java

```diff
@@ -60,28 +53,34 @@ public void showTotalCost(String dollarCost) {
         Command addOrderCommand = new AddOrderCommand(orderChickenRice, ui, pantry, orderList, menu);
         addOrderCommand.execute();
 
-        String actualOutput = String.join(",", commandOutput);
+        String actualOutput = baos.toString().trim();
+        System.setOut(originalOut);
 
-        String expectedOutput = Messages.CHEF_MESSAGE
-                + Messages.LINE_STRING
+        String expectedOutput = "I'm busy crafting your selected dish in the virtual kitchen of your dreams. "
+                + "Bon appétit!"
+                + "-----------------------------------------------------"
```

Comment:
Perhaps you should use the Strings stored in Messages.java instead of "magic strings"